src/Uno.UWP/Storage/StorageFile.cs

```diff
@@ -205,9 +205,12 @@ public async Task MoveAsync(CancellationToken ct, StorageFolder targetFolder, st
 			}
 
 			_fileUri = new Uri("file://" + targetPath);
-		}
+		}
+
+		public IAsyncOperation<BasicProperties> GetBasicPropertiesAsync()
+			=> GetBasicPropertiesAsyncTask(new CancellationToken()).AsAsyncOperation<BasicProperties>();
 
-		public async Task<BasicProperties> GetBasicPropertiesAsync(CancellationToken ct)
+		public async Task<BasicProperties> GetBasicPropertiesAsyncTask(CancellationToken ct)
```

Comment:
Why did you create this method ?

Reply:
Because tooltip in VStudio, in my app, shows that `GetBasicPropertiesAsync() is not implemented in  Uno`  (from Generated); and if I would remove it in Generated, it would appear as missing method, so I included it in implementation (as redirection to previously written (by someone else) method returning Task<>).

Reply:
GetBasicPropertiesAsync had a parameter, you should be able to keep it that way, even with adding  `IAsyncOperation<BasicProperties> GetBasicPropertiesAsync()`

Reply:
Microsoft documentation:
public IAsyncOperation<BasicProperties> GetBasicPropertiesAsync()
StorageFile.cs implemented:
public async Task<BasicProperties> GetBasicPropertiesAsync(CancellationToken ct)

So, my task here was to 'convert'
a) parameterless  GetBasicPropertiesAsync() to same with parameter, and 
b) IAsyncOperation<BasicProperties> to Task<BasicProperties>

Reply:
There is no conversion possible here. You have to keep `public async Task<BasicProperties> GetBasicPropertiesAsync(CancellationToken ct)`, you can keep `IAsyncOperation<BasicProperties> GetBasicPropertiesAsync()`.